### PR TITLE
update pua alerts

### PR DESCRIPTION
**Pull request by roaaattalla**

### CODE REVIEW

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 1]
```diff
@@ -3189,6 +3189,39 @@ action.cyences_notable_event_action.system_compromised_drilldown = `cs_sophos` s
 action.cyences_send_email_action = 1
 action.cyences_notable_event_action.products = Sophos
 
+[Sophos - Potentialy Unwated Application PUA]
```
Comment (hardikhdholariya) [inline cyences_app_for_splunk/default/savedsearches.conf:3192]: Title should be **[Sophos - Failed to CleanUp Potentialy Unwated Application by Sophos]**

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 2]
```diff
@@ -3189,6 +3189,39 @@ action.cyences_notable_event_action.system_compromised_drilldown = `cs_sophos` s
...
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule = 2,17,32,47 * * * *
+description = This alert will trigger when a Sophos endpoint detects potential unwanted application . \
```
Comment (hardikhdholariya) [inline cyences_app_for_splunk/default/savedsearches.conf:3202]: Update the description accordingly.

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 3]
```diff
@@ -3189,6 +3189,39 @@ action.cyences_notable_event_action.system_compromised_drilldown = `cs_sophos` s
...
+dispatch.earliest_time = -17m@m
+dispatch.latest_time = -2m@m
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search =`cs_sophos` sourcetype="sophos_events" type="Event::Endpoint::CorePuaCleanFailed" \
+| stats count, latest(_time) as _time, values(name) as threat, values(source_info.ip) as src_ip, values(user) as user by host, location | sort -count \
+| eval cyences_severity = "critical" \
```
Comment (hardikhdholariya) [inline cyences_app_for_splunk/default/savedsearches.conf:3214]: Sophos alert has "medium" severity. Just confirm with the SOC team and update accordingly.

**Review by hardikhdholariya — CHANGES_REQUESTED** [review]

File: cyences_app_for_splunk/default/savedsearches.conf (hunk 1 above)
Comment (VatsalJagani) [reply cyences_app_for_splunk/default/savedsearches.conf:3192]: Potentially
Unwanted

**Review by hardikhdholariya — APPROVED** [review]

**Pull request merged**